explorer/ast/declaration.cpp

```diff
@@ -368,7 +368,9 @@ auto FunctionDeclaration::Create(Nonnull<Arena*> arena,
 }
 
 void CallableDeclaration::PrintDepth(int depth, llvm::raw_ostream& out) const {
-  out << "fn " << name_ << " ";
+  if (auto name = GetName(*this)) {
+    out << "fn " << *name << " ";
+  }
```

Comment:
Do we want to change this to `CARBON_CHECK` the name as well?

Reply:
That'd be fair for consistency.